CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 * Add support for `secretPrefix` property for User Operator to prefix all secret names created from KafkaUser resource.
 * Allow configuring labels and annotations for Cluster CA certificate secrets
+* Add `sasl.jaas.config` to generated secrets for KafkaUser with SCRAM-SHA-512 authentication.
```

Comment:
It's better to not use just the field name but explaining what it is, so for example.
"Add the JAAS configuration string in the `sasl.jaas.config` property to the generated secrets for KafkaUser with SCRAM-SHA-512 authentication"